gap/ColouredComplexes/isoscelesColouring.gi

```diff
@@ -477,37 +481,25 @@ InstallMethod( LocalSymmetryOfEdges,
         [IsEdgeColouredTwistedPolygonalComplex], 
         [IsIsoscelesColouredSurface], 0 );
 
-
-#######################################
-##
-##      AllIsosceles...
-##
-
-## Surface-variation
-
-
-
-## Surface-variation
-InstallOtherMethod( AllIsoscelesColouredSurfaces, "for a simplicial surface",
-    [IsSimplicialSurface],
-    function(simpSurf)
-        return AllIsoscelesColouredSurfaces(simpSurf, []);
-    end
-);
-
 ################################
 ##
-#+ AllIsocelesColouredSurfaces( <surf> ) . . . find all isosceles colourings
+## AllIsocelesColouredSurfaces( <surf> ) . . . find all isosceles colourings
 ##
 ## This function determines all edge colourings of a simplicial surface such
 ## that two edges have the same colour, representing isosceles triangles. Note
 ## that not all isosceles colourings are obtained by setting two colours
 ## equal in a wild colouring.
 ##
-
 InstallOtherMethod( AllIsoscelesColouredSurfaces, "for a simplicial surface",
-    [IsSimplicialSurface,IsList],
-            function( surf, li )
```

Comment:
what was the purpose of li in the old version of the method?

Reply:
Since li wasn't use in the old implementation, I think there wasn't a purpose.

Reply:
ah ok that's funny :)! thank you